src/client/pages/FinancieleHulp/FinancieleHulp.tsx

```diff
@@ -17,7 +17,7 @@ import styles from './FinancieleHulp.module.scss';
 
 const DISPLAY_PROPS = {
   title: 'Status',
-  to: 'Bekijk op FiBu',
+  to: 'Bekijk op Krefia',
```

Comment:
Klopt dit wel? De naam Krefia gebruiken we nergens anders dan hier. Moet het niet een distinctie zijn tussen 'Bekijk op Kredietbank' en 'Bekijk op FIBU' ?

Reply:
Ik zal het even checken met Micha, het staat zo wel in het RD